var/spack/repos/builtin/packages/py-sip/package.py

```diff
@@ -53,3 +56,11 @@ def build(self, spec, prefix):
 
     def install(self, spec, prefix):
         make('install')
+
+    @run_after('install')
+    def extend_path_setup(self):
+        if 'module=PyQt5.sip' in self.spec:
```

Comment:
Is this still needed? Is it needed for other `module` options too?

Reply:
Yep, it is still needed since  there is no `__init__.py` under `PyQtX` folders. Just pushed an improved version where we also account for PyQt4 case.